src/js/media-time-display.js

```diff
@@ -50,7 +55,12 @@ const DEFAULT_MISSING_TIME_PHRASE = 'video not loaded, unknown time.';
 const updateAriaValueText = (el) => {
   const currentTime = el.mediaCurrentTime;
   const [, seekableEnd] = el.mediaSeekable ?? [];
-  const endTime = el.mediaDuration || seekableEnd;
+  let endTime;
```

Comment:
We check for `endTime == null` later on, which makes me feel like this should now either default to null, or we should check for `undefined` or `!endTime` later.